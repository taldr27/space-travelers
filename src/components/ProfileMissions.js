import PropTypes from 'prop-types';

const ProfileMissions = ({ name }) => (
  <>
    <tr>
      <td>{name}</td>
    </tr>
  </>
);

ProfileMissions.propTypes = {
  name: PropTypes.string.isRequired,
};

export default ProfileMissions;
