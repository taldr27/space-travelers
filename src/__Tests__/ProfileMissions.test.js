import React from 'react';
import { render } from '@testing-library/react';
import ProfileMissions from '../components/ProfileMissions';

describe('Profile Missions Testing', () => {
  it('Should render Profile Missions and compare with Snapshot', () => {
    const { asFragment } = render(<ProfileMissions name="Thaicom" />);
    expect(asFragment(<ProfileMissions name="Thaicom" />)).toMatchSnapshot();
  });
});
