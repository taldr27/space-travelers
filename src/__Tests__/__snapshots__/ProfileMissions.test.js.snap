// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Missions Testing Should render Profile Missions and compare with Snapshot 1`] = `
<DocumentFragment>
  Thaicom
</DocumentFragment>
`;
